nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -808,7 +808,18 @@ export function RanEdit(props: Props) {
       setError(e.response?.data?.message ?? e.message);
     }
   };
-
+  const isEnodebUnregistered = (enb: enodeb, currentGateway: lte_gateway) => {
```

Comment:
Fix the flow error on this line. Asides from running flow on your machine, you can see the result of the Github check running `nms-flow-test`